pkg/scale/varying_data_type.go

```diff
@@ -43,7 +44,7 @@ func mustNewVaryingDataTypeSliceAndSet(vdt VaryingDataType,
 	values ...VaryingDataTypeValue) (vdts VaryingDataTypeSlice) {
 	vdts = NewVaryingDataTypeSlice(vdt)
 	if err := vdts.Add(values...); err != nil {
-		panic(err)
+		panic(fmt.Sprintf("adding VaryingDataTypeValue: %s", err))
```

Comment:
```suggestion
		panic(fmt.Sprintf("adding varying data type value: %s", err))
```
try to make errors consistent. For example, have the same error wrapper for everytime `vdts.Add` fails.

Generally, we try to have extra meaning in the error than the code. So, `VaryingDataTypeValue` becomes `varying data type value` in error messages